src/en/installconfig-server-iso.md

```diff
@@ -1,73 +1,102 @@
+[here](https://help.ubuntu.com/lts/serverguide/installing-from-cd.html)
+for details on installing a generic Ubuntu Server.
 
-The region controller installation will ask for a username followed by a password
-for the MAAS administrator. If these are not provided, an administrator account
-will need to be create manually before you can use MAAS. 
 
-![rack controller install](./media/iso-install-rack_01.png)
+## Choosing region controller
 
-The rack controller installation requires no further MAAS-specific input, and
-the Ubuntu Server installation will proceed normally:
+After having chosen to install a region controller, three consecutive dialog
+boxes will appear in order to do the following:
 
-Enter a username and password to create a user account for the server. This
-account is separate from the administrator account created for the MAAS Region
-Controller. You will then be asked whether you wish to encrypt the home
-directory and asked for the server's time zone. 
+- Install a MAAS environment
+- Set a username for the MAAS administrator
+- Set a password for the MAAS administrator
 
-The next stage configures storage and partitions before the system software will
-start to be installed. After this has completed you get the option to define an
-HTTP proxy server, which may be needed for your MAAS server to access the
-internet, and how you want updates delivered. The choice is between 'No
-automatic updates', 'Install security updates automatically' and 'Manage system
-with Landscape'. 
+^# Walkthrough of MAAS environment installation
 
-![rack updates](./media/iso-install-region-updates.png)
+  Confirm the installation of a MAAS environment.
+  ![region controller install dialog 1](./media/iso-install-region_01.png)
+  
+  Provide the name of the initial MAAS administrator. It can be anything but in
+  this example 'admin' was chosen. This account is distinct from the system
+  account that will be created later as part of the generic Ubuntu Server install.
+  ![region controller install dialog 2](./media/iso-install-region_02.png)
+  
+  Complete the creation of the MAAS administrator account by supplying a password.
+  ![region controller install dialog 3](./media/iso-install-region_03.png)
 
-!!! Note: [Landscape](https://landscape.canonical.com) is Canonical's leading
-system management tool.
+During one step of the generic Server install you will be asked whether a proxy
+is needed for the server itself. This is separate from the
+[proxy feature](./installconfig-proxy.html) that MAAS provides to its nodes.
 
-It is important to set updates appropriately so you will receive timely
-updates of the MAAS server software, as well as other essential services that
-may run on this server.
 
-The final two steps install the Grub boot loader and set the time. You will
-then be notified that installation is complete and you should restart the
-server.
+## Choosing rack controller
```

Comment:
I find this subheading and 'Choosing region controller' a little unclear, especially if you're browsing the sub-headings rather than following the ISO install. Maybe remove 'Choosing' (which to me makes it sound like a choice and makes me want to add an 'a')?